src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -128,17 +130,27 @@ public void canSimulateApiUserFlow() throws URISyntaxException {
             apiUser.id,
             apiUser.email,
             TEMP_AUTH0_USER_PASSWORD);
-        HttpResponse<String> getTokenResponse = mockAuthenticatedPost(endpoint,
+        HttpResponse<String> getTokenResponse = mockAuthenticatedRequest(endpoint,
+            "",
             apiUser,
-            ""
+            HttpUtils.REQUEST_METHOD.POST,
+            false
         );
         LOG.info(getTokenResponse.body());
         assertEquals(HttpStatus.OK_200, getTokenResponse.statusCode());
+        TokenHolder tokenHolder = JsonUtils.getPOJOFromJSON(getTokenResponse.body(), TokenHolder.class);
+
+        // Define the header values to be used in requests from this point forward.
+        HashMap<String, String> headers = new HashMap<>();
+        headers.put("Authorization", "Bearer " + tokenHolder.getAccessToken());
+        headers.put("x-api-key", apiUser.apiKeys.get(0).value);
```

Comment:
Is the x-api-key header still needed (since we've removed the code that reads that value and finds the ApiUser)?